### PR TITLE
README - update toggle parameters

### PR DESCRIPTION
**Pull request by tak2siva**

### CODE REVIEW

File: install/kubernetes/helm/istio/README.md [hunk 1]
```diff
@@ -160,12 +160,12 @@ Helm charts expose configuration options which are currently in alpha.  The curr
 | `global.arch.s390x` | Specifies the scheduling policy for `s390x` architectures | 0 = never, 1 = least preferred, 2 = no preference, 3 = most preferred | `2` |
 | `global.arch.ppc64le` | Specifies the scheduling policy for `ppc64le` architectures | 0 = never, 1 = least preferred, 2 = no preference, 3 = most preferred | `2` |
 | `ingress.enabled` | Specifies whether Ingress should be installed | true/false | `true` |
-| `gateways.istio-ingressgateway.enabled` | Specifies whether Ingress gateway should be installed | true/false | `true` |
-| `gateways.istio-egressgateway.enabled` | Specifies whether Egress gateway should be installed | true/false | `true` |
+| `gateways.enabled` | Specifies whether gateway should be installed | true/false | `true` |
```
Comment (ymesika) [inline install/kubernetes/helm/istio/README.md]: Don't delete those. They are still available to toggle either ingress or egress.
E.g. one might only want to install the ingressgateway.

Comment (tak2siva) [reply install/kubernetes/helm/istio/README.md]: @ymesika Are you sure ? I couldn't find those variables !

Comment (tak2siva) [reply install/kubernetes/helm/istio/README.md]: My bad! Updated the README again please check.

**Comment by linsun** [comment]
/lgtm

note: in the future, it is best to submit these corrections to 1.1 release branch.

**Comment by ymesika** [comment]
I'm closing this one because of #10032 which will eventually be merged to `master`.

**Pull request closed without merging**